# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: toolkit/components/printing/tests/file_link_modulepreload.html
# Commit: 635806b9356c
# Full Hash: 635806b9356cd535ff079a4cfceb9ca954ac23da
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2022-07-11 15:47:29
# Description:
#   Bug 1777496 - Check if ModuleLoader exists before calling SetAcquiringImportMaps. r=smaug
#   
#   For the print preview documents, they are cloned from the original document in
#   Document::CreateStaticClone[1]. But the cloned document isn't attched to any
#   nsIContentViewer yet so it won't have any global object. The document is
# ==============================================================================

diff -r d83579bec018 -r 635806b9356c toolkit/components/printing/tests/file_link_modulepreload.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/toolkit/components/printing/tests/file_link_modulepreload.html	Mon Jul 11 09:38:55 2022 +0000
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<html>
+  <head>
+    <link rel="modulepreload" href="test.js">
+  </head>
+  <body>
+    <script type="importmap">
+      {
+        "imports": {
+          "test": "./test.js"
+        }
+      }
+    </script>
+    <p>Hello world</p>
+  </body>
+</html>
